Dockerfile

```diff
@@ -12,12 +12,12 @@ RUN apt-get update \
   && apt-get -y install curl \
   && apt-get clean \
   && rm -rf /var/apt/lists/*
-RUN curl -fsSL https://deb.nodesource.com/setup_14.x | bash -
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
 RUN apt-get install -y nodejs \
   && apt-get clean \
   && rm -rf /var/apt/lists/*
 
-ENV MONGODB_URI mongodb://localhost/5e-database
+ENV MONGODB_URI mongodb://localhost:27017/5e-database
```

Comment:
not certain if this change is necessary, but it works on the machine i was testing on